frontend/components/organization/Edit.jsx

```diff
@@ -125,6 +143,15 @@ function Edit({ actions }) {
             alertRole={false}
           />
           )}
+        { mostRecentlyAddedUser
+          && (
+          <AlertBanner
```

Comment:
Do we want anyway to clear the alert banner? Depending on the user's screen size and position if the form, the alert may be painted out of view. Do we want to do a modal or trap the focus with scroll to it so they see the alert for sure?

Reply:
@sknep and I talked about that. There's already also the brief alert at upper right which shows up regardless of where the user is scrolled. The point of having this one stay is that users (at least the one user who opened the associated ticket) want to be able to copy the URL, and if it goes away that's going to be challenging.

Reply:
Thanks for walking through the thought process. It makes sense.

Reply:
Sure thing! The whole interaction could still use more work in the future. In the discussion on #4108 Sarah suggested alternate alerts to be shown when a user has been added to the org but no invitation was sent because they were already a Pages user, or when no invitation was sent because the user specified is already a member of the org, but differentiating those cases (or back end errors) will require other changes, and we didn't think it needed to hold this up.